README.md

```diff
@@ -14,6 +14,9 @@ I am building a blog list application using MERN stack, that allows users to sav
 -   Run the command for Development Environment: `npm run dev`
 -   Run the command for testing Environment: `npm run test` or simply `npm test`
 
+## Exposed Endpoints
+There are currently *three* restful endpoints exposed for public consumption. You can read more about them [here](ENDPOINTS.md).
+
```

Comment:
Moving the detailed information into a secondary file really helps to keep the README focused on user consumption of the application, developers and those that are curious can and will navigate to the documentation through the hyperlink.